src/main/java/org/ggp/base/util/game/LocalGameRepository.java

```diff
@@ -226,7 +226,10 @@ private static int getMaxVersionForDirectory(File theDir) {
                 } else if (theFile.getName().endsWith(".png")) {
                     // TODO: Handle other binary formats?
                     return readBinaryFile(theFile);
-                } else if (theFile.getName().endsWith(".xsl")) {
+                } else if (theFile.getName().endsWith(".jpg")) {
+                    return readBinaryFile(theFile);
+                } else if (theFile.getName().endsWith(".xsl"))
+                {

```

Comment:
That was a mistake on my part.  My personal settings for Intellij are for a different indentation style and I didn't correct it before checking it in.